src/domain/entities/LoggerConfig.ts

```diff
@@ -0,0 +1,30 @@
+import { Id } from "./Base";
+
+interface LoggerConfigBase {
+    debug?: boolean;
+}
+
+export type LoggerConfig = ProgramLoggerConfig | ConsoleLoggerConfig;
+
+export interface ConsoleLoggerConfig extends LoggerConfigBase {
+    type: "console";
+}
+
+export interface ProgramLoggerConfig extends LoggerConfigBase {
+    type: "program";
+    baseUrl: string;
+    auth?: Auth;
+    programId: Id;
+    loggerDataElements: LoggerDataElements;
+    organisationUnitId?: Id;
```

Comment:
best practices, subjective: In my experience, it's better not to have optional properties for types we control. This way we must be explicit if we want to keep them empty. Instead we can use Maybe:  `organisationUnitId: Maybe<Id>;`